gradle/init-scripts/mirrors-init.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * This is an init script for internal usage at Gradle Inc.
+ */
+allprojects {
+    buildscript {
+        repositories {
+            maven {
+                name "gradle-mirror"
+                url "http://dev12.gradle.org:8081/artifactory/gradle-repo/"
```

Comment:
Ideally this would also be taken from the parameter that we already have on teamcity, just like the RepoScriptBlockutil does.

Reply:
I've changed the init script so that the URL is loaded from the environmental variable.